File: JavaLab/src/main/java/ch2/section2/Quiz2.java
package ch2.section2;

public class Quiz2 {
    public static void main(String[] args) {
        int month = 8;

        switch (month){
            case 3:
            case 4:
            case 5:
                System.out.println("봄.....");
                break;
            case 6:
            case 7:
            case 8:
                System.out.println("여름....");
                break;
            case 9:
            case 10:
            case 11:
                System.out.println("가을...");
            case 12:
            case 1:
            case 2:
                System.out.println("겨울....");
            default:
                System.out.println("1~12 사이의 정수값이어야 합니다.");
        }
    }
}

File: KotlinLab/KotlinLab/src/main/kotlin/ch2/section4_control/q4/Quiz2.kt
package org.example.ch2.sec4_control

import java.text.SimpleDateFormat

fun saySeason(data: String): String {
    val date = SimpleDateFormat("yyyy-MM-dd").parse(data)
    return when(date.month + 1) {
        12, 1, 2 -> "겨울이군요"
        3, 4, 5 -> "봄이군요"
        6, 7, 8 -> "여름이군요"
        9, 10, 11 ->"가을이군요"
        else -> "잘못된 날짜 형식이에요"
    }
}

fun main() {
    println(saySeason("2025-04-30"))
    println(saySeason("2025-12-30"))
}